extensions/wikia/UserProfilePageV3/UserProfilePageController.class.php

```diff
@@ -491,6 +491,8 @@ public function saveUsersAvatar( $userId = null, $data = null ) {
 			switch ( $data->source ) {
 				case 'sample':
 					$user->setOption( 'avatar', $data->file );
+					// remove old avatar file
+					Masthead::newFromUser($user)->removeFile(true);
```

Comment:
The guidelines require spaces before and after function parameter.  Also, you can leave off `true` here since that's the default value for the `$addLog` parameter of `removeFile`.